### DISCUSSION TITLE
How to use raytune in my own improved yolov8 models？

### DISCUSSION THREAD

**Discussion by CyberJZTXJ** [ultralytics/ultralytics#15854] [Q&A]
i add Irmb moudle to improve my yolov8model，but it report AttributeError: **Can't get attribute 'iRMB' on <module 'ultralytics.nn.modules.block' from '/home/usr/.local/lib/python3.9/site-packages/ultralytics/nn/modules/block.py'>**，it seems that ray tune do not use my modified model，it use the official model installed by pip，how to solve the problems？

**Comment by glenn-jocher** [comment]
@CyberJZTXJ to use Ray Tune with your custom YOLOv8 model, ensure your modified model is correctly imported and referenced in your script. You may need to adjust your environment to point to your custom model instead of the one installed via pip. For detailed guidance, refer to the [Ray Tune integration documentation](https://docs.ultralytics.com/integrations/ray-tune/).
